.github/workflows/publish-conda.yml

```diff
@@ -0,0 +1,20 @@
+name: Publish to Conda
+
+on:
+  # Triggers the workflow on a new release
+  release:
+    types: [created]
+    
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v1
+    - name: publish-to-conda
+      uses: MichaelsJP/conda-package-publish-action@v1.0.0
+      with:
+        subDir: 'conda'
+        AnacondaToken: ${{ secrets.ANACONDA_TOKEN }}

```

Comment:
I added a secret with a  token I got from https://anaconda.org/.

Reply:
@alexgarel Great, I already tested it with my fork and it seems to build and upload!

Reply:
So, on a new release creation - it should automatically upload to PyPI and conda as well